src/components/Header/Header.js

```diff
@@ -153,60 +160,63 @@ const Header = ({location}) => {
   }
 
   return (
-    <Menu attached size="large">
-      <Container>
-        <Menu.Item>
-          <img src={logo} alt="Kids First logo" />
-        </Menu.Item>
-        <Menu.Item header as={NavLink} to="/" activeClassName="">
-          Data Tracker
-        </Menu.Item>
-        {loggedIn && !location.pathname.includes('/versions/') && (
-          <>
-            <Menu.Item as={Nav} to="/study" content="Studies" />
-            {hasPermission(profile, 'view_settings') && (
-              <Menu.Item as={Nav} to="/releases/history">
-                Releases
-                <Label content="beta" color="blue" attached="bottom right" />
-              </Menu.Item>
-            )}
-            <Menu.Menu position="right">
-              {hasPermission(profile, 'add_referraltoken') && (
-                <AddUserButton profile={profile} />
+    <Container fluid>
+      <AppBanner />
```

Comment:
Does it make more sense to put it below the main navigation or above?

Reply:
Let me try both to see what it looks like